src/test/resources/org/prebid/server/it/test-application.properties

```diff
@@ -1,5 +1,3 @@
-adapters.generic.enabled=true
```

Comment:
Changes like this should be in another PRs. Pls add only bidder specific change and remove others